aim/db/model_base.py

```diff
@@ -130,10 +130,26 @@ def to_attr(self, session):
         Child classes should override this method to specify a custom
         mapping of model properties to resource attributes.
         """
-        return {k: self.get_attr(session, k) for k in dir(self)
-                if (not k.startswith('_') and
+        # "object_dict" and "tree uses LargeBinary sqlalchemy datatype.
+        # LargeBinary sqlalchemy datatype needs "bytes-like" object
+        # And in Py2, string are bytes-like objects while in Py3 they aren't.
+        # So we need to store "bytes-like" object in DB objects.
+        # For getting the resource attr dict from model object
+        # we need to decode to native string (for Py3 compatibility)
```

Comment:
nit: missing '.' at the end of the setence.

Reply:
sure, will edit this.